gateway/src/apicast/policy.lua

```diff
@@ -14,6 +14,8 @@ local PHASES = {
     'content', 'balancer',
     'header_filter', 'body_filter',
     'post_action',  'log', 'metrics',
+
+    'ssl_certificate',
```

Comment:
Can you add this new phase to the list that we have in `policies.md` ?

Reply:
Yep.